### PR TITLE
feat: set autoDeleteOnStop using the tasks command

### PR DESCRIPTION
**Pull request by RobotHanzo**
### Motivation
`tasks task <name> set` was missing the ability to edit `autoDeleteOnStop`

### Modification
Added the command `tasks task <name> set autoDeleteOnStop <enabled>`

### Result
`tasks task <name> set` can now modify `autoDeleteOnStop`, without having to edit the json file by hand

##### Other context
N/A

### CODE REVIEW

**Comment by 0utplay** [comment]
I'm not really sure. When I implemented the commands for v4 I left that one out on purpose because a lot of users do net get what this means and confuse it with `staticServices`. At least until there is no proper documentation I would leave it like it is

**Comment by RobotHanzo** [comment]
> I'm not really sure. When I implemented the commands for v4 I left that one out on purpose because a lot of users do net get what this means and confuse it with `staticServices`. At least until there is no proper documentation I would leave it like it is

I thought leaving it out from tasks setup would be enough (as this is probably what most starters start with)
Advanced users probably need a convenient method to configure this though (in my opinion)

**Comment by RobotHanzo** [comment]
> I'm not really sure. When I implemented the commands for v4 I left that one out on purpose because a lot of users do net get what this means and confuse it with `staticServices`. At least until there is no proper documentation I would leave it like it is

I thought leaving it out from tasks setup would be enough (as this is probably what most starters start with)
Advanced users probably need a convenient method to configure this though (in my opinion)

File: node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java [hunk 1]
```diff
@@ -385,6 +385,21 @@ public void renameTask(
...
+  @CommandMethod("tasks task <name> set autoDeleteOnStop <enabled>")
+  public void setAutoDeleteOnStop(
+    @NonNull CommandSource source,
+    @NonNull @Argument("name") Collection<ServiceTask> tasks,
+    @Argument("enabled") @Liberal boolean enabled
+  ) {
+    this.applyChange(
+      source,
+      tasks,
+      ServiceTask.Builder::maintenance,
```
Comment (0utplay) [inline node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java]: I do not think this is right here

**Review by 0utplay — CHANGES_REQUESTED** [review]

File: node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java (hunk 1 above)
Comment (RobotHanzo) [reply node/src/main/java/eu/cloudnetservice/node/command/sub/TasksCommand.java]: > I do not think this is right here

Forgot to change it, fixed

**Review by 0utplay — APPROVED** [review]

**Pull request merged**